src/main/assets/js/courts.ts

```diff
@@ -0,0 +1,65 @@
+import $ from 'jquery';
+import {orderToggleState} from '../../enums/searchToggleState';
+import {CourtsTableSearch} from './courts-table-search';
+
+export class CourtsController {
+
+  private contentId = '#main-content';
+  private tableContainerId = '#tableContainer';
+  private toggleClosedCourtsDisplay = 'toggleClosedCourtsDisplay';
+  private searchCourtsFilter = 'searchCourts';
+  private courtsNameAscToggleId = '#courtsNameAscToggle';
+  private courtsUpdatedAscToggleId = '#courtsUpdatedAscToggle';
+  private tableCourtsNameId = '#tableCourtsName';
+  private tableCourtsUpdatedId = '#tableCourtsUpdated';
+
+  constructor() {
+    this.initialize();
+  }
+
+  private initialize(): void {
+    $(() => {
+      if($(this.tableContainerId).length) { // so that this doesn't load on each page
+        CourtsTableSearch.setUpTable('name', true);
+        this.setUpToggleClosedCourtsDisplay();
+        this.setUpCourtsDynamicSearchFilter();
+        this.setUpAscDecNameFilter();
+        this.setUpAscDecUpdatedDateFilter();
+      }
+    });
+  }
+
+  private setUpToggleClosedCourtsDisplay(): void {
+    $(this.contentId).on('change', `input[name=${this.toggleClosedCourtsDisplay}]`, e => {
+      e.preventDefault();
+      $(this.courtsUpdatedAscToggleId).val() == orderToggleState.INACTIVE ?
```

Comment:
Minor point - I think putting the ? and : on separate lines makes it more readable.